qiskit/algorithms/eigensolvers/vqd.py

```diff
@@ -94,15 +94,15 @@ class VQD(VariationalAlgorithm, Eigensolver):
                 overlap estimation as indicated in the VQD paper.
             ansatz (QuantumCircuit): A parameterized circuit used as ansatz for the wave function.
             optimizer(Optimizer | Sequence[Optimizer]): A classical optimizer or a list of optimizers,
-            one for every k-th eigenvalue. Can either be a Qiskit optimizer or a callable
+                one for every k-th eigenvalue. Can either be a Qiskit optimizer or a callable
                 that takes an array as input and returns a Qiskit or SciPy optimization result.
             k (int): the number of eigenvalues to return. Returns the lowest k eigenvalues.
             betas (list[float]): Beta parameters in the VQD paper.
                 Should have length k - 1, with k the number of excited states.
                 These hyper-parameters balance the contribution of each overlap term to the cost
                 function and have a default value computed as the mean square sum of the
                 coefficients of the observable.
-            initial point (Sequence[float] | Sequence[Sequence[float]] | None): An optional initial
+            initial_point (Sequence[float] | Sequence[Sequence[float]] | None): An optional initial
```

Comment:
Now that the initial point's attribute name is correct, sphinx complains that it shows up both here and in `Args`. I cannot leave an inline comment but https://dev.azure.com/qiskit-ci/qiskit/_build/results?buildId=55980&view=logs&j=a15d9aca-d75c-55b1-7f76-889cfe884cf0&t=154f6391-7dd7-5733-2230-2f10f0798b76 can be fixed by removing lines 145-149.

Reply:
@ElePT this was a bad change. It's removing actual code that will impact runtime behavior. The better fix is to delete this line of the docs and not mess with the code.